README.rst

```diff
@@ -302,6 +302,20 @@ supported by the SDK. You can still use these endpoints by using the ``make_requ
 The ``Client`` class and Box objects have a ``get_url`` method. Pass it an endpoint
 to get the correct URL for use with that object and endpoint.
 
+For API calls which require a body, ``make_request()`` accepts ``**kwargs`` after ``method`` and ``url``.
+
+.. code-block:: python
+
+    # https://developer.box.com/reference#update-terms-of-service-user-status
+    # Updates a user's ToS status
+
+    # JSONify the body
+    body = json.dumps({"is_accepted":true})
+
+    # Pass body as "data" argument
+    data = body
```

Comment:
Why do we set `data = body` here?

Reply:
@anthonywee That seems to be what make_request needs to make passing data as body work properly. One of the most recent messages in our internal slack channel for the Python SDK refers to setting data = body, or data = whatever object contains the jsonified data. Maybe there's a better way to communicate that?

Reply:
sorry, that's my personal ^ account. This is from my actual box github account.